sql/_13_issues/_23_1h/answers/cbrd_24598.answer

```diff
@@ -0,0 +1,32 @@
+===================================================
+0
+===================================================
+decode( ?:0 , '', c, null, c, -1)    
+-1     
+
+===================================================
+decode( ?:0 , '', c, null, c, -1)    
+-1     
+
+===================================================
+decode( ?:0 , '', c, null, c, -1)    
+-1     
```

Comment:
An error occurs and it fails.
Shouldn't the error code be included in the answer?

Reply:
The answer sheet is written with the expected result.
Now the query executing order problem is detected as a known bug.
So this will be detected as a fail and verified as a known bug.
Until the bug is solved we will track this as a failure case in the regression test.

Reply:
Fixing this bug will take longer than expected.

Until then, the sql test will continue to fail.
As a result, development teams can't merge code.

What I said I want to check through fail is not to keep it as fail.

Right now, it makes an answer with the actual result, and expects this TC to fail when this bug is fixed.
Again, if it continues to fail, it will be difficult to merge the code.

Reply:
I missed the circle CI sql test. 
The answer sheet is revised to occur errors and it will be revised after the known bug is solved.